pkg/bootstrap/bootstrap.go

```diff
@@ -3,61 +3,134 @@ package bootstrap
 import (
 	"context"
 	"crypto/rand"
+	"errors"
 	"fmt"
 	"net/http"
 	"os"
 	"strings"
 
+	"github.com/gptscript-ai/go-gptscript"
 	types2 "github.com/obot-platform/obot/apiclient/types"
 	"github.com/obot-platform/obot/pkg/api"
 	"github.com/obot-platform/obot/pkg/api/authz"
 	"github.com/obot-platform/obot/pkg/gateway/client"
-	"github.com/obot-platform/obot/pkg/gateway/server/dispatcher"
 	"github.com/obot-platform/obot/pkg/gateway/types"
-	"github.com/obot-platform/obot/pkg/system"
 	"k8s.io/apiserver/pkg/authentication/authenticator"
 	"k8s.io/apiserver/pkg/authentication/user"
 )
 
-const bootstrapCookie = "obot-bootstrap"
+const (
+	bootstrapCookie            = "obot-bootstrap"
+	bootstrapCredential        = "obot-bootstrap"
+	bootstrapCredentialContext = "obot-bootstrap"
```

Comment:
Having three constants with the same name seems strange. Can we have one constant with a reasonable name to use for all three?